apps/website/blog/2024-02-28-maci-v1.2.0.md

```diff
@@ -104,7 +104,7 @@ With every issue, PR, feature and roadmap iteration, we welcome feedback to ensu
 
 For those looking to contribute directly, report bugs, or offer feedback, our [GitHub repository](https://github.com/privacy-scaling-explorations/maci) is open for issues and discussions. We're eager to assist with your projects or contributions.
 
-For practical implementation insights, review our docs as well as the [clr.fund](https://github.com/clrfund/monorepo/) and [QF](https://github.com/privacy-scaling-explorations/qf) repositories as reference implementations. Both are quadratic funding implementations, a mechanism which otherwise is highly susceptible to collusion and bribery. Most notably, clr.fund is already working on integrating MACI v1.2.0, after having used v0.x until now. You can follow their development effort under this [GitHub branch](https://github.com/clrfund/monorepo/tree/feat/maci-v1).
+For practical implementation insights, review our docs as well as the [clr.fund](https://github.com/clrfund/monorepo/) and [QF](https://github.com/quadratic-gardens/qfi) repositories as reference implementations. Both are quadratic funding implementations, a mechanism which otherwise is highly susceptible to collusion and bribery. Most notably, clr.fund is already working on integrating MACI v1.2.0, after having used v0.x until now. You can follow their development effort under this [GitHub branch](https://github.com/clrfund/monorepo/tree/feat/maci-v1).
```

Comment:
I still see 404 in the `Github branch` link (https://github.com/clrfund/monorepo/tree/feat/maci-v1)

Reply:
missed that sorry - but I think that's fine, at the time we were redirecting to the branch were they were implementing MACI v1, now it's deleted and we should not be changing a blogpost content aside from fixing links so we can probably keep like that